File: frontend/src/itsme/itsme.js
/*
As we don't really use Itsme, we simulate the calls here.
*/

export function getItsmeToken(username) {
    return Array.from(username).map((c, i) => String.fromCharCode((c.charCodeAt(0) + i) % 128)).join("");
}

export function getItsmeName(token) {
    return Array.from(token).map((c, i) => String.fromCharCode((c.charCodeAt(0) - i) % 128)).join("");
}

File: backend/itsme/itsme.py

# As we don't really use Itsme, we simulate the calls here.

def get_itsme_token(username):
    return "".join([chr((ord(c)+i % 128)) for i,c in enumerate(username)])


def get_itsme_name(token):
    return "".join([chr((ord(c)-i % 128)) for i,c in enumerate(token)])
